src/main/java/se/liu/ida/hefquin/engine/wrappers/graphqlwrapper/GraphQLSolutionGraphSolver.java

```diff
@@ -0,0 +1,20 @@
+package se.liu.ida.hefquin.engine.wrappers.graphqlwrapper;
+
+import java.util.List;
+
+import org.apache.jena.rdf.model.Model;
+
+import se.liu.ida.hefquin.engine.data.SolutionMapping;
+import se.liu.ida.hefquin.engine.query.SPARQLQuery;
+
+public interface GraphQLSolutionGraphSolver {
+    
+    /**
+     * Queries @param solutionGraph using the SELECT query @param query and
+     * @return a list of corresponding solution mappings.
+     * @throws RunTimeException if for some reason the execution of the query over the
+     * given solutionGraph did not work correctly.
```

Comment:
Please don't throw runtime exceptions. Instead, throw ordinary exceptions (create special classes for them); that way, these exceptions need to be explicitly declared and the compiler complains if they are not caught. Hence, when using this code somewhere else, one is immediately aware of the exceptions and has to take them into account when coding. In contrast, runtime exceptions have the tendency to be forgotten to be handled.